app/code/Magento/ProductAlert/Controller/Add/Price.php

```diff
@@ -6,16 +6,18 @@
 
 namespace Magento\ProductAlert\Controller\Add;
 
-use Magento\Framework\App\Action\HttpGetActionInterface;
-use Magento\ProductAlert\Controller\Add as AddController;
-use Magento\Framework\App\Action\Context;
-use Magento\Customer\Model\Session as CustomerSession;
-use Magento\Store\Model\StoreManagerInterface;
 use Magento\Catalog\Api\ProductRepositoryInterface;
-use Magento\Framework\UrlInterface;
+use Magento\Customer\Model\Session as CustomerSession;
```

Comment:
Could you replace `\Magento\Customer\Model\Session` to `CustomerSession` in all places, for instance in constructor's phpdoc block

The same with all other imported classes

Reply:
**done**